rust/zemu/include/zemu_ui_s.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#define KEY_SIZE (17 + 1)
```

Comment:
This header (along with the X one) is currently unused as I have moved the code from these headers to view.h by hand to speed up since I was struggling getting these headers in the compilation